tests/test_cli.py

```diff
@@ -226,6 +226,21 @@ def test_matched(self, script_runner: ScriptRunner) -> None:
         assert result.success
         assert result.stdout[0].isdigit(), "Output does not start with a number"
 
+    @pytest.mark.vcr
+    def test_fields(self, script_runner: ScriptRunner) -> None:
+        args = [
+            "stac-client",
+            "search",
+            STAC_URLS["EARTH-SEARCH"],
+            "-c",
+            "cop-dem-glo-30",
+            "--max-items",
+            "1",
+        ]
```

Comment:
I think the fields got dropped from the test, but they're in the cassette (#vcr-problems). This fails:

```shell
pytest tests/test_cli.py::TestCLISearch::test_fields --record-mode=rewrite
```

Reply:
yes sorry was ensuring that it failed and forgot to put them back 🤦🏻